template/llama2-chinese.yaml

```diff
@@ -0,0 +1,145 @@
+apiVersion: template.app.sealos.io/v1beta1
+kind: Template
+metadata:
+  name: llama2-chinese
+spec:
+  title: 'llama2-chinese'
+  url: 'https://github.com/LinkSoul-AI/Chinese-Llama-2-7b'
+  github: 'https://github.com/luanshaotong/text-generation-webui-docker.git'
+  author: 'sealos'
+  description: 'Fully open-source and commercially usable Chinese version of the Llama2 model, along with bilingual Chinese-English SFT dataset. Input format strictly adheres to the llama-2-chat format, ensuring compatibility and adaptation with all optimizations designed for the original llama-2-chat model.'
+  readme: 'https://github.com/luanshaotong/text-generation-webui-docker/blob/main/README.md'
```

Comment:
Replace with `raw.githubusercontent.com` to get direct access.

Reply:
Done